### PR TITLE
[Discussion] Change bright colors to old version

### PR DESCRIPTION
**Pull request by dritter**
This should fix the second issue in #955 that @DuncanHuang brought up.

The colors were changed in `v0.6.5`. This PR updates the colors to the previous version.

So the colors `red`, `green`, `yellow`, `blue`, `magenta`, `cyan` and `white` point the bright colors:
```
'black'   '000'
'red'     '001'
'green'   '002'
'yellow'  '003'
'blue'    '004'
'magenta' '005'
'cyan'    '006'
'white'   '007'
```
I did a separate PR for this, because this needs a bit of discussion. The [page](https://jonasjacek.github.io/colors/) that was the source for the color names lists the colors mentioned above in the range of 8-15, like they are in `v.0.6.5`. But that means that users who upgrade may have to change their color definitions. That is why we need to discuss this here. The commit to look at is 8e966e3636ce7c7162b60763088252b7111985e5

In short:
- Option A
Change colors like in this PR. This means that users who upgrade P9K from an old Version (pre `v0.6.5`) get the _same_ colors as before, but if they upgrade from `v0.6.5`, colors change. Second downside is that we turn away from other color definitions.

- Option B
Leave colors as they are. This means that users who upgrade P9K from an old Version (pre `v0.6.5`) get _different_ colors.

### CODE REVIEW

**Comment by AuroraWright** [comment]
The new color scheme looks terrible on Solarized Dark (on the macOS Terminal.app at least). Colors are muted and dark-ish, and I basically need to customize most blocks (including the "dir" one) for them to stand out.

**Comment by bhilburn** [comment]
@dritter - Thanks so much for putting this together, explaining the situation, and laying out the options. I would like to revert back to the previous colors; breaking existing configs is never a good plan, IMO.

It looks like this PR was filed before I had actually merged the v066 staging branch into master, so there is a bit of git logistics to go through, here. I'll get that sorted and then get this branch merged. =)

@AuroraWright - Thanks for your input here, by the way!
